api/models/metadata.go

```diff
@@ -64,28 +77,33 @@ func (metadata *Metadata) Scan(value interface{}) error {
 
 func (metadata *Metadata) ToLabel() map[string]string {
 	labels := map[string]string{
-		labelOrchestratorName: "merlin",
-		labelAppName:          metadata.App,
-		labelEnvironment:      metadata.Environment,
-		labelStreamName:       metadata.Stream,
-		labelTeamName:         metadata.Team,
+		prefix + LabelAppName:          metadata.App,
```

Comment:
does user need to explictily add `/` in the prefix?

Reply:
Yes, so it's up to user to set their own delimiter or not at all.

Reply:
but the latest changes, we enforce user to add `/` on their prefix, since prefix "gojek.com" won't pass the validation

Reply:
Correct. We've added the validation as you suggested when initializing the prefix to guard this enforcement.